core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -155,6 +155,13 @@ public class TableWriter implements Closeable {
     private final WeakClosableObjectPool<MemoryCMOR> walColumnMemoryPool;
     private final ObjList<MemoryCMOR> walMappedColumns = new ObjList<>();
     private final IntList symbolRewriteMap = new IntList();
+    private MemoryCMR detachedMetaMem;
```

Comment:
please rename members along "attach" and "detach" workflows

Reply:
All these members are used on detached partition in `attach` methods prior to attaching.
I think naming them `attachedMetaMem` or `attachingMetaMem` and alike will be more confusing than what it is now.

Reply:
I renamed to `attachMetaMem` and etc, it doesn't sound too bad